app/src/main/res/values/settings_keys.xml

```diff
@@ -172,6 +172,7 @@
     <string name="show_memory_leaks_key" translatable="false">show_memory_leaks_key</string>
     <string name="allow_disposed_exceptions_key" translatable="false">allow_disposed_exceptions_key</string>
     <string name="show_original_time_ago_key" translatable="false">show_original_time_ago_text_key</string>
+    <string name="disable_media_tunneling_key" translatable="false">disable_media_tunneling_text_key</string>
```

Comment:
What's the reason for using a different value here?
```suggestion
    <string name="disable_media_tunneling_key" translatable="false">disable_media_tunneling_key</string>
```

Reply:
was asking that myself. I was just copying from  `<string name="show_original_time_ago_key" translatable="false">show_original_time_ago_text_key</string>`

Reply:
I have removed it from both

Reply:
if you change the key value, we might want to add a SettingsMigration. I am  not sure if this a bit too much effort for a rarely used debug setting.

Reply:
yes, I would say that a migration is not required. But out of curiosity, is there a concept for it in Android and/or do we already have a settings migration in NewPipe?

Reply:
We already have one: [SettingsMigrations.java](https://github.com/TeamNewPipe/NewPipe/blob/dev/app/src/main/java/org/schabi/newpipe/settings/SettingMigrations.java)